exonum/src/node/state.rs

```diff
@@ -863,6 +920,38 @@ impl State {
         }
     }
 
+    /// Finds unknown transactions in the block.
+    pub fn unknown_block_txs(
```

Comment:
The method name is misleading, as it mutates the state. I'd suggest something like `create_incomplete_block`.